lib/temporal/connection/grpc.rb

```diff
@@ -31,6 +31,12 @@ class GRPC
         max_page_size: 100
       }.freeze
 
+      # Temporal's other clients have to override grpc's default client-side payload limit of 4mb.
+      # Java uses 25mb, go uses 64mb. Most temporal API responses aren't large, but
+      # PollWorkflowTaskQueueResponse contains a workflow's full history, which can be large. We
+      # match the golang sdk just because it's bigger.
+      MAX_RECEIVED_MESSAGE_LENGTH = 64 * 1024 * 1024
```

Comment:
@jeffschoner-stripe is this used anywhere?

Reply:
It is not. It must be a remnant from our internal fork that I assume was already upstreamed. I'll remove it for now, but should bring this back in a separate PR. It's part of a change that sets the default to be the same as the Go SDK. The limit is lower by default in the Ruby GRPC stack, but I don't recall exactly how much smaller.